code/mathematical_algorithms/src/collatz_conjecture_sequence/collatz_conjecture_sequence.py

```diff
@@ -0,0 +1,11 @@
+def collatzSequence(n):
+    while n != 1:
+        if n%2 == 0:
+            n = int(n/2)
+        else:
+            n = int((3 * n) + 1)
```

Comment:
you could cast to integer in the beginning of the function, if you want to make sure that **n** is integer.